lib/spack/spack/build_systems/python.py

```diff
@@ -160,11 +160,16 @@ def build_directory(self):
     def python(self, *args, **kwargs):
         inspect.getmodule(self).python(*args, **kwargs)
 
+    def setup_args(self, spec, prefix):
```

Comment:
The `python setup.py -h` help message calls these "global options". How about:
```suggestion
    def global_args(self, spec, prefix):
```